contrib/src/main/java/com/datatorrent/contrib/kafka/AbstractKafkaInputOperator.java

```diff
@@ -203,6 +204,14 @@ public AbstractKafkaInputOperator()
    */
   protected abstract void emitTuple(Message message);
```

Comment:
That's why I'm asking can we do that. It does have compatibility issue but on the other hand if some one wants to emit the tuple along with partition id etc, he/she is forced to implement the emitTuple method as well. Well, we can keep it as it is now. @DT-Priyanka can you file a jira to make this change in 0.9 kafka input operator.  We can change the existing interface for that one